model/marketplace_plugin.go

```diff
@@ -90,6 +91,7 @@ func (filter *MarketplacePluginFilter) ApplyToURL(u *url.URL) {
 	}
 	q.Add("filter", filter.Filter)
 	q.Add("server_version", filter.ServerVersion)
+	q.Add("license_sku_short_name", filter.LicenseSkuShortName)
```

Comment:
0/5 if the URL value should be omitted if `filter.LicenseSkuShortName` is empty.